### DISCUSSION TITLE
How to use mp4 videos in Next Js

### DISCUSSION THREAD

**Discussion by saikrishnabanda5** [GoogleChrome/lighthouse#14507] [Q&A]
Can anyone help me with this?
![Screenshot from 2022-11-07 21-19-35](https://user-images.githubusercontent.com/54619513/200353953-fbf2349b-d633-4092-8b21-22a3e0197f36.png)

**Comment by adamraine** [comment]
https://web.dev/total-byte-weight/#how-to-reduce-payload-size
